### ISSUE TITLE
Add AccessLint for automated web accessibility testing

### ISSUE THREAD

**Issue by jbampton** [mruby/mruby.github.io#98]
Setup a free plan for open source.

AccessLint brings automated web accessibility testing into your development workflow. When a pull request is opened, AccessLint reviews the changes and comments with any new accessibility issues, giving you quick, timely, and targeted feedback, before code goes live.

AccessLint helps you keep on target with digital accessibility by giving specific, ongoing accessibility feedback to your team. It can catch simple errors that might break your website for your customers with disabilities.

Accessibility guidelines defined in WCAG 2.0 help us build applications that work better for more people. They also give us a rough measure of compliance with legal requirements outlined in the ADA and Section 508. AccessLint runs a growing list of WCAG 2.0 tests, e.g. missing alt attributes that make pages more opaque for screenreader users, positive tabindex values that can create confusion for keyboard users, invalid ARIA, and unlabelled fields.

https://github.com/marketplace/accesslint
